app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +11,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.MyWeatherApp">
-        <activity android:name=".MainActivity">
+        <activity
+            android:name=".MainActivity"
+            android:configChanges="orientation|screenSize">
```

Comment:
why do you need this? I think that you don't need it, please remove it and add the screenOrientation property if you need it

Reply:
Done, thx, now i erased it, and used it "screenOrientation"